attributes/default.rb

```diff
@@ -185,11 +193,24 @@
     default['cfncluster']['rhel']['extra_repo'] = 'rhui-REGION-rhel-server-optional' if node['platform'] == 'redhat' && node['platform_version'].to_i >= 7
 
   when 'amazon'
-    default['cfncluster']['base_packages'] = %w[vim ksh tcsh zsh openssl-devel ncurses-devel pam-devel net-tools openmotif-devel
-                                                libXmu-devel hwloc-devel db4-devel tcl-devel automake autoconf pyparted libtool
-                                                httpd boost-devel redhat-lsb mlocate mpich-devel R atlas-devel fftw-devel
-                                                libffi-devel dkms mysql-devel libedit-devel postgresql-devel postgresql-server
-                                                sendmail cmake byacc libglvnd-devel mdadm libgcrypt-devel]
+    case node['platform_version'].to_i
+    when 2
+      default['cfncluster']['base_packages'] = %w[vim ksh tcsh zsh openssl-devel ncurses-devel pam-devel net-tools openmotif-devel
+                                                  hwloc-devel libdb-devel libxml2-devel libgcrypt-devel boost-devel perl-devel
+                                                  dpkg-dev tar gzip python-pip]
```

Comment:
is ```python-pip``` really needed?
Also let's double check the list, because for example I see mdadm missing, which should be needed for RAID

Reply:
I tried to keep this list as minimal as possible. I only included a package if it was needed to execute compilation recipes or run integration tests. I believe python-pip was needed, but i'll need to check for which package.

For mdam or other missisng packages, this set was sufficient for building AMIs and successfully running all of the integration tests on those built AMIs.

Reply:
The following are installed as part of the first provisioner to run when building the AMIs:
```
autoconf
automake
bison
flex
gcc
gcc-c++
libtool
patch
rpm-build
rpm-sign
system-rpm-config
byacc
cscope
ctags
diffstat
doxygen
elfutils
gcc-gfortran
git
indent
intltool
patchutils
rcs
subversion
swig
systemtap
curl
install
jq
wget
```

so those should probably be in this list to handle the use of non-pcluster AMIs.

Reply:
python-pip wasn't needed as it turns out. i've removed it.

Reply:
After trying to start a cluster with the modified cookbook, it looks like python-pip is actually required by the python_package resource used to install the batch CLI. Adding it back now.